xmppserver/src/main/java/org/jivesoftware/openfire/ldap/LdapManager.java

```diff
@@ -377,6 +381,12 @@ public LdapManager(Map<String, String> properties) {
         }
         groupSearchFilter = properties.get("ldap.groupSearchFilter");
 
+        flattenNestedGroups = false;
+        String flattenNestedGroupsStr = properties.get("ldap.flattenNestedGroups");
+        if (flattenNestedGroupsStr != null) {
```

Comment:
Nitpicky, but no need for the null check as Boolean.valueOf does the right thing in this case.

Reply:
Yes you are right. But all other boolean properties are handled like this. E.g `"ldap.debugEnabled"` or `"ldap.sslEnabled"`. So it is just consistent when it is done like this.